dhall-try/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <title>The Dhall configuration language</title>
-    <meta name="description" content="A non-repetitive alternative to YAML">
+    <meta name="description" content="Maintainable configuration files">
```

Comment:
"files" doesn't add much IMHO.

```suggestion
    <meta name="description" content="Maintainable configuration">
```

Reply:
I would keep the "files" because there are other ways a program's configuration can be supplied, such as:

* Command-line arguments
* Environment variables
* Settings in a database

Also, if I search for "Configuration" in Google I don't get as many relevant hits as I do when searching for "Configuration file".  Indeed, the relevant Wikipedia page is named ["Configuration file"](https://en.wikipedia.org/wiki/Configuration_file) whereas ["Configuration"](https://en.wikipedia.org/wiki/Configuration) takes you to a disambiguation page.